src/components/tag/filter-tag.ts

```diff
@@ -23,7 +24,22 @@ const { prefix } = settings;
  * Filter tag.
  */
 @customElement(`${prefix}-filter-tag`)
-export default class BXFilterTag extends FocusMixin(LitElement) {
+export default class BXFilterTag extends HostListenerMixin(FocusMixin(LitElement)) {
+  @query('button')
+  protected _buttonNode!: HTMLButtonElement;
+
+  /**
+   * Handles `click` event on this element.
+   * @param event The event.
+   */
+  @HostListener('shadowRoot:click')
+  // @ts-ignore: The decorator refers to this method but TS thinks this method is not referred to
+  private _handleClick = (event: MouseEvent) => {
+    if (event.composedPath().indexOf(this._buttonNode!) >= 0) {
```

Comment:
```suggestion
    if (this.disabled && event.composedPath().indexOf(this._buttonNode!) >= 0) {
```

Just in case this makes my above comment clearer. Thanks again for your fix @emyarod!